SeaPkg/Library/BasePeCoffValidationLib/BasePeCoffValidationLib.c

```diff
@@ -414,7 +414,22 @@ PeCoffImageValidationContent (
   }
 
   if (CompareMem ((UINT8 *)TargetImage + Hdr->Offset, ContentHdr->TargetContent, Hdr->Size) != 0) {
-    DEBUG ((DEBUG_ERROR, "%a: Current entry range 0x%p: 0x%x does not match input content at 0x%p\n", __func__, ContentHdr, Hdr->Size, ContentHdr->TargetContent));
+    DEBUG ((DEBUG_ERROR, "%a: Current entry 0x%p: Content mismatch in image at address 0x%p\n", __func__, ContentHdr, ContentHdr->TargetContent));
+
+    DEBUG ((DEBUG_ERROR, "%a: Expected content: ", __func__));

```

Comment:
Would it be better to use DUMP_HEX here?

Reply:
DUMP_HEX comes with some additional bloat I did not like for the purpose of showing a just a few bytes. Mainly it includes the current line you are on, and the ascii display, which I did not like.

The line looks like this:

`13:52:40.030 :     00000000: 4D 5A 00 00 00 00 00 00-00 00 00 00 00 00 00 00  *MZ..............*`

I'd be happy to switch it if you prefer.

Reply:
I have a slight preference for reusiing DUMP_HEX, but I think either could work here.

Reply:
I'm going to keep it as is unless @kuqin12 has a strong preference. I will leave this comment open, for him to close, or respond back telling me to change it :)